wandb/sdk/artifacts/artifact.py

```diff
@@ -79,37 +79,27 @@
 class Artifact:
     """Flexible and lightweight building block for dataset and model versioning.
 
-    Constructs an empty artifact whose contents can be populated using its `add` family
-    of functions. Once the artifact has all the desired files, you can call
+    Construct an empty W&B Artifact. Populate an artifacts contents with methods that
+    begin with `add`. Once the artifact has all the desired files, you can call
     `wandb.log_artifact()` to log it.
 
     Arguments:
-        name: A human-readable name for this artifact, which is how you can identify
-            this artifact in the UI or reference it in `use_artifact` calls. Names can
+        name: A human-readable name for the artifact. Use the name to identify
+            a specific artifact in the W&B App UI. You can interactively reference
```

Comment:
```suggestion
            a specific artifact in the W&B App UI or programatically. You can interactively reference
```
the name will also be used as an argument for some SDK methods as well